### PR TITLE
Hoodies at JScon Budapest 

### PR DESCRIPTION
**Pull request by pawan92**
Created a blog post on JS conference that recently took place in Budapest!

### CODE REVIEW

File: _drafts/JSCON Budapest [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+
+title: 'JS CON  Budapest'
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: Can we change this to 'JSConf Budapest'? As that is the actual title of the conference.

File: _drafts/JSCON Budapest [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+author: Pawan
+layout: blog-post
+comments: true
+
+---
+
+Hey Everyone!
+
+The  Hoodies  (Jan, Lena, Nick, Nolan)  were  just  at  JSConf   Budapest  this   week ! It   was an exciting event with lots of cool stuff happening!
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: Lots of extra double spacing in this sentence that needs to be tidied up :)

File: _drafts/JSCON Budapest [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+
+# JSCON Budapest
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: Can we change this to `JSConf Budapest` ? :)

File: _drafts/JSCON Budapest [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+
+Situated in Budapest, this year's JavaScript conference showcased plenty of workshops and engaging speakers.
+Budapest is a city located on the Danube river. The capital consists of two cities, Buda and Pest.Budapest has been
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: Can we put a space between `Bude and Pest.` an `Budapest` please?

File: _drafts/JSCON Budapest [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+reported as one of the best places to visit in Europe. As such it makes for a perfect place to  hold the JS conference.
+
+#Speakers
+1.LAURIE VOSS @seldo
+Laurie talked about how  NPM  expects you to use NPM. He also talked about the commands and workflows that can make you better at NPM.
+The talk can be found here :https://youtu.be/zWEU8kNKi3Q
+2.Suz Hinton @noopat
+Suze talked about  how Data creation, analysis, and corruption can be paired withJavaScript to discover new and improved things. The
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: needs to be a space in `withJavaScript`

File: _drafts/JSCON Budapest [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+The  Hoodies  (Jan, Lena, Nick, Nolan)  were  just  at  JSConf   Budapest  this   week ! It   was an exciting event with lots of cool stuff happening!
+
+# JSCON Budapest
+
+Situated in Budapest, this year's JavaScript conference showcased plenty of workshops and engaging speakers.
+Budapest is a city located on the Danube river. The capital consists of two cities, Buda and Pest.Budapest has been
+reported as one of the best places to visit in Europe. As such it makes for a perfect place to  hold the JS conference.
+
+#Speakers
+1.LAURIE VOSS @seldo
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: Some of the names are in ALL CAPS, some are not. Could we make sure each speakers name is written in the same way?

File: _drafts/JSCON Budapest [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+3.DENYS MISHUNOV @mishunov
+Denys talked about the importance of performance in his talk. There are a lot of factors influencing performance  but Denys talked about the importance of perception.
+Although the site may have few requests and be mathematically fast, it nonetheless may seem slow. He gives an interesting
+speech on the how the user's perspective is the true measure of speed. Check it out here!
+https://youtu.be/j9TrSkLZa5A?list=PL37ZVnwpeshE6PbF5GB4hvtU3A4HqMfxf
+4. Yan Zhu @bcrypt
+Yan talked about the importance of security and how it shouldn't be compromised for performance. Check it out here:
+https://youtu.be/n9Fe68xgKUw?list=PL37ZVnwpeshE6PbF5GB4hvtU3A4HqMfxf
+5.TimPietrusky @TimPietrusky/ MartinSchuhfuss @usefulthink
+Tim and Martin put on a awesome audio/visual  performance with their skills in music,lights,MDI,and electronic.
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: "a awesome" should be "an awesome"

File: _drafts/JSCON Budapest [hunk 8]
```diff
@@ -0,0 +1,122 @@
...
+Lena (our very own!!)  talks about how software plays an important role in peoples' everyday lives. She also dives into how you as a person can make a change in the tech industry.
+You can listen to her talk here:
+7. Sam Bellen
+Sam decided to use the web-audio-api to change the sounds of his guitar . He talks  about how he accomplished this and showcases his skills.
+Check it out here:https://youtu.be/BEdFcnI-ppk
+8.Stefan Baumgartner @ddprrt
+Stefan talks about the new version of HTTP and how it will affect the javascript developers of the future.
+Watch his talk here: https://youtu.be/98z0XjYWX0o
+9.Rachel  White @ohhoe
+Rachel had a problem. Her cats kept waking her up at ungodly hours. So she built a web based feeder that is remote controlled!
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: can we find a better word than "ungodly" here? ☺️

File: _drafts/JSCON Budapest [hunk 9]
```diff
@@ -0,0 +1,122 @@
...
+stored  and how we can start using it for ourselves in our React apps.
+16.Oliver Joseph Ash @OliverJAsh
+What if we are working on an app and  we lose internet connection? In this talk Oliver explains using service workers to build an offline page for
+theguardian.com . Working offline soon won't be an issue anymore!
+17.Safia Abdalla  @captainsafia
+Safia cracks some funny jokes and takes us on an adventure on memory management in javascript.Through this Safia tells us how we can implement code
+better to improve performance and learn javascript concepts better.
+Check out her interesting talk here: https://youtu.be/AeUCN2lPqL8
+18. Nick Hehr @hipsterbrown
+Last but not least our very own Nick does a fantastic job describing how the community is as important as the code. Its the people behind the project
```
Comment (varjmes) [inline _drafts/JSCON Budapest]: "Its" should be "It's"

**Comment by varjmes** [comment]
Hey pawan!

Thanks so much for this.
There are a few spelling changes we need to make here, I commented on where I think these should happen. Also there is a lot of inconsistent spacing, as well as the speaker names varying in format.

Let us know if you need help with this, I am sure other editorial members can help too 💃 

Thanks again! Sounds like a great conference ☺️

**Comment by jennwrites** [comment]
Hey @pawan92 thanks for writing this up. After reviewing this, I suggest that you try to get a hold of the Hoodies who attended (@janl, @nolanlawson or @HipsterBrown) for a quote on the event. 

I would cut the rest of the content (the synopsis of each talk, unless recommended or referred to by one of the people mentioned about). The focus of this blog post should be on the Hoodies representing our awesome project at a great event. Don't worry about length ☺️ just try to get some great quotes from our teammates! 

If you have questions please DM me! Thanks again 🐶

**Comment by HipsterBrown** [comment]
There are some good pull quotes from the mood video :smile: -> https://www.youtube.com/watch?v=mJoS_pLbiWc

**Comment by nolanlawson** [comment]
If you're looking for pull quotes on JSConf BP, here's some things I would say about it:

I had a blast. My workshop on Progressive Web Apps drew a nice little crowd (~30 people), who were all eager to learn about how to make offline web apps with modern web tools. In the course of the 6-hour workshop, we built a small offline Star Wars app (here's [the finished product](http://nolanlawson.github.io/star-wars-offline/)), and afterwards we went out for drinks and mingled with other speakers and organizers. I found the pre-conference workshop to be a great way for speakers to get to know the local developer community.

Flaki, Mozilla, and all the organizers did a fantastic job running the event, and really took care of their speakers and workshoppers. (I felt like royalty when they picked me up at the airport!) The venue itself was hip and relaxed, and people stayed late into the evening for the drinks, atmosphere, and sparkling post-conf conversation. 

Oh yeah, and the speakers (that's what we were all there to see, right? 😉) were phenomenal! All in all, a great conference and highly recommended.

**Comment by pawan92** [comment]
hey @renrutnnej @Charlotteis i have taken out the synopsis and put in some quotes from the suggested youtube video. i can add more quotes if necessary let me know :)

**Comment by jennwrites** [comment]
Hey @pawan92,

First off, thank you so much for volunteering to do this write up, and putting all of the effort into it that you did. 🙏🏻 I am so grateful for your willingness and enthusiasm to share what the Hoodies are doing around the world, with the world. 

I'm making the decision not to publish this post for a couple of reasons. The first being timeliness, which is 100% my fault. The conference was 4 months ago, and you volunteered to take this on 3 months ago, and then I didn't get around to giving you feedback in a timely manner whatsoever. I'm so sorry for that. 

Secondly, I didn't accurately explain the type of post I was hoping for. If you check out [Hoodie at OSCON in Austin](http://hood.ie/blog/hoodie-at-oscon-austin-2016.html) on the blog, this is the type of synopsis with photos I had been envisioning. You couldn't have known that though, based on the [news issue I created](https://github.com/hoodiehq/editorial/issues/91).

You did exactly what I asked and I'm so thankful for you being such an awesome contributor. I'm hoping to create more opportunities for editorial contributors to get involved very soon, so I hope this experience doesn't discourage you from participating in the future. 

Please let me know if you have any questions as I am happy to discuss. Thanks again @pawan92.

**Pull request closed without merging**